### PR TITLE
Update Readme with new features(#8) 

### PR DESCRIPTION
**Pull request by charlypro123**
publish version 1.2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,42 +5,66 @@
 
 <img src="icon.png" width="20%"/>
 
-Picoleto is the gentleman that will validate your Spanish identity documents.
+Picoleto is the gentleman that will validate your Spanish identity documents(NIF,NIE,Passport),Spanish Phones(MobileLine and LandLine) and Spanish ZipCodes.
```
Comment (alorma) [inline README.md]: Maybe add a list of identity types here?

```markdown
Picoleto is the gentleman that will validate your Spanish documents:
* Identity 
    * NIF
    * NIE
    * Passport
* Phones
   * MobileLine
   * LandLine
* ZipCodes
```

Comment (EsteveAguilera) [reply README.md]: Sounds good @alorma ... what do you think @charlypro123?

Comment (charlypro123) [reply README.md]: Yes,it become more understandable.

Comment (EsteveAguilera) [reply README.md]: Great, go ahead then! :tada:

File: README.md [hunk 2]
```diff
@@ -5,42 +5,81 @@
...
+    -NIE
+    -Passport
+
+-Phones
+
+    -MobileLine
+    -LandLine
+
+-ZipCodes
+
```
Comment (EsteveAguilera) [inline README.md]: The indented lines appears as code blocks, what does not looks really good... can you check it out?

Comment (charlypro123) [reply README.md]: Fixed!

**Comment by EsteveAguilera** [comment]
Looks good to me... @alorma what do you think?

**Review by alorma — APPROVED** [review]
LGTM

**Comment by EsteveAguilera** [comment]
Great work @charlypro123 and thanks for the feedback @alorma !

**Pull request merged**